src/lib/select/select.spec.ts

```diff
@@ -882,10 +882,10 @@ describe('MatSelect', () => {
           expect(options[2].getAttribute('role')).toEqual('option');
         }));
 
-        it('should set aria-selected on each option', fakeAsync(() => {
-          expect(options[0].getAttribute('aria-selected')).toEqual('false');
-          expect(options[1].getAttribute('aria-selected')).toEqual('false');
-          expect(options[2].getAttribute('aria-selected')).toEqual('false');
+        it('should set aria-selected on each option for single select',  fakeAsync(() => {
+          expect(options[0].getAttribute('aria-selected')).toEqual(null);
+          expect(options[1].getAttribute('aria-selected')).toEqual(null);
+          expect(options[2].getAttribute('aria-selected')).toEqual(null);
```

Comment:
Instead of doing `getAttribute` with `null`, can you test `hasAttribute` with `false` for the omission case? We also have been trying to include expanded assertion messages lately, like this:
```ts
expect(options[0].hasAttribute('aria-selected'))
    .toBe(false, 'Expected unselected, single-select options not to have aria-selected set');
```

Reply:
It can be reduced further like this: 

```ts
expect(options.every(option => !option.hasAttribute('aria-selected'))).toBe(true, 'Expected none of the options to have aria-selected);
```